workflow-core/src/main/java/com/squareup/workflow/WorkflowAction.kt

```diff
@@ -57,6 +55,12 @@ interface WorkflowAction<StateT, out OutputT : Any> {
     nextState = mutator.state
   }
 
+  @Suppress("DEPRECATION")
+  @Deprecated("Implement Updater.apply")
+  fun Mutator<StateT>.apply(): OutputT? {
+    throw UnsupportedOperationException()
+  }
+
```

Comment:
Moved this method below the non-deprecated one so that the non-deprecated one is what gets resolved for `WorkflowAction.apply` links in kdoc.